### DISCUSSION TITLE
Tips for OP crosh

### DISCUSSION THREAD

**Discussion by doofenator** [3kh0/ext-remover#771] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
1. if you need any other extension other than the blocker(clever, ublock o), just use second version
2. You can use your phone hotspot, if you have one, instead of wifi as you can quickly switch off your hotspot
3. Use a seperate/home computer to reference the commands if you need
4. If you don't have a home pc, copy the op crosh commands onto a usb's text file
5. Don't be stupid

**Comment by thecoolmiker** [comment]
Op crosh doesnt work for me since my crosh was blocked by my school. Even when i do this sadly doesnt work. 😔
